LMGeocoderDemo/LMGeocoder/LMViewController.m

```diff
@@ -119,6 +126,21 @@ - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray
                                         }];
 }
 
+- (NSString *)cacheFilePath {
+
+    NSString *documentDir = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) firstObject];
+    return [documentDir stringByAppendingPathComponent:@"abcdef"];
+}
+
+- (void)cacheResults:(NSArray *)results {
+    [NSKeyedArchiver archiveRootObject:results toFile:[self cacheFilePath]];
+}
+
+- (void)loadCached {
+    NSArray *locations = [[NSKeyedUnarchiver unarchiveObjectWithFile:[self cacheFilePath]] mutableCopy];
+    NSLog(@"%@", locations);
```

Comment:
Is there need to keep the logging?

Reply:
Yes, its just for testing, 
It's a demo project not related to the lib it self